pkgs/tools/misc/hackertyper/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, ncurses }:
+
+stdenv.mkDerivation rec {
+  name = "hackertyper";
+  version = "b24e682ac291b0e5bed11a6871efa3c97e72dfb6";
+
+  src = fetchFromGitHub {
+    owner  = "Hurricane996";
+    repo   = "Hackertyper";
+    rev    = "${version}";
+    sha256 = "14f7pfjp28iqy0hby50sdx602zva4g3d0ndvwzs03qq262rgd6n4";
+  };
+
+
+  makeFlags = [ "PREFIX=$(out)" ];
+  buildInputs = [ ncurses ];
+
+  preInstall = ''
+    mkdir -p $out/bin
+    mkdir -p $out/share/man/man1
+  '';
+
+
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    $out/bin/hackertyper -v
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A rewrite of hackertyper.net";
```

Comment:
Missing homepage, probably link to github unless they have another website linked somewhere